docs/content/preview/api/ysql/the-sql-language/statements/cmd_copy.md

```diff
@@ -152,3 +152,8 @@ yugabyte=# COPY large_table FROM '/home/yuga/Desktop/large_table.csv'
 - If the table does not exist, errors are raised.
 - `COPY TO` can only be used with regular tables.
 - `COPY FROM` can be used with tables, foreign tables, and views.
+
+Additionally, the following copy options may be useful for speeding up copying, or allowing for faster recovery from a partial state:
+* `DISABLE_FK_CHECK` will skip foreign key check when copying new rows to the table.
+* `REPLACE`: if the primary / unique key of the new row conflicts with that of the existing row, the new row will replace the existing row in the table.
+* `SKIP n`: skips the first `n` rows of the file. Require `n` to be nonnegative.
```

Comment:
The format looks not consistent. Some options are followed with `:`, while some are not. We should follow the same pattern.